contents/docs/libraries/js/index.mdx

```diff
@@ -456,6 +456,158 @@ const handleCookieConsent = (consent) => {
 
 - If you don't want PostHog to store anything on the user's browser (e.g. if you want to rely on your own identification mechanism only or want completely anonymous users), you can set `disable_persistence: true` in PostHog's config. If you do this, remember to call [`posthog.identify`](#identifying-users) **every time** your app loads. If you don't, every page refresh is treated as a new and different user.
 
+## Amending events before they are captured
+
+When initializing the SDK, you can provide a `before_send` function.  This can be used to amend or reject events before they are sent to PostHog. 
```

Comment:
```suggestion
Since version 1.187.0, when initializing the SDK, you can provide a `before_send` function.  This can be used to amend or reject events before they are sent to PostHog. 
```